Zend/zend_partial.stub.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/** @generate-class-entries */
+
+final class Partial extends Closure
+{
+    private function __construct() {}
+
+    public static function bind(
+        Closure $closure,
+        ?object $newThis,
+        object|string|null $newScope = "static"
+    ): ?Closure {}
+
+    public function bindTo(?object $newThis, object|string|null $newScope = "static"): ?Closure {}
```

Comment:
```suggestion
    public function bindTo(?object $newThis, object|string|null $newScope = "static"): ?Partial {}
```
?

Reply:
The methods aren't (yet) implemented, and there's a bug in arg info stuff internally that prohibits me declaring it that way.